torch/nn/modules/batchnorm.py

```diff
@@ -245,9 +250,13 @@ def _check_input_dim(self, input):
 
 
 class LazyBatchNorm1d(_LazyBatchNorm):
-    r"""A :class:`torch.nn.BatchNorm1d` module with lazy initialization of
-    the ``num_features`` argument of the :class:`BatchNorm1d` that is inferred
-    from the ``input.size(1)``.
+    r"""A :class:`torch.nn.BatchNorm1d` module with lazy initialization.
+
+    In this module, the `weight` and `bias` are of :class:`torch.nn.UninitializedParameter`
+    class. And the `running_mean` and `running_var` are of :class:`torch.nn.UnintializedBuffer`.
+    They will be initialized after the first call to ``forward`` is done and the
```

Comment:
There is no need to explain that it change after the forward here. This is already explained in the common lazy doc.
Similarly no need to mention the exact types, you can just say that `weight`, `bias`, `running_mean` and `running_var` will be lazily initialized.
And then say explicitly that the num_features will be inferred from the input size.

Maybe something like:
A :class:`torch.nn.BatchNorm1d` module with lazy initialization of
the ``num_features`` argument of the :class:`BatchNorm1d` that is inferred
from the ``input.size(1)``.
The attributes that will be lazily initialized are `weight`, `bias`, `running_mean` and `running_var`.

Reply:
updated.